### ISSUE TITLE
review alternative links

### ISSUE THREAD

**Issue by ngosang** [Jackett/Jackett#10586]
@garfield69 @ilike2burnthing 
Some of them with error 503 are protected by CloudFlare but they are valid. Maybe we have to add FlareSolverr info.
Some of them with error 502 are blocked in my country but they are valid.

- [x] 1337x  ||  https://1337x.to/  ||  Bad HTTP response code: 502
- [x] 1337x  ||  https://1337x.gd/  ||  Bad HTTP response code: 503
- [x] 1337x  ||  https://1337x.is/  ||  Bad HTTP response code: 503
- [x] 1337x  ||  https://1337x.st/  ||  Bad HTTP response code: 503
- [x] 1337x  ||  https://x1337x.ws/  ||  Bad HTTP response code: 503
- [x] 1337x  ||  https://x1337x.eu/  ||  Bad HTTP response code: 503
- [x] 1337x  ||  https://x1337x.se/  ||  Bad HTTP response code: 503
- [x] 1337x  ||  https://1337.root.yt/  ||  Bad HTTP response code: 301
- [x] 1337x  ||  https://1337x.unblocked.rest/  ||  Bad HTTP response code: 504
- [x] 3evils  ||  https://www.3evils.net/  ||  HTTPSConnectionPool(host='www.3evils.net', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9259a1ba00>: Failed to establish a new connection: [Errno 111] Connection refused'))
- [x] anidex  ||  https://anidex.info/  ||  Bad HTTP response code: 403
- [x] ast4u  ||  https://www.ast4u.me/  ||  HTTPSConnectionPool(host='www.ast4u.me', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7f92585a5820>, 'Connection to www.ast4u.me timed out. (connect timeout=60)'))
- [x] audiobookbay  ||  https://audiobookbay.unblockit.dev/  ||  Bad HTTP response code: 503
- [x] bigtower  ||  http://bigtower.info/  ||  HTTPConnectionPool(host='bigtower.info', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f92585a5250>, 'Connection to bigtower.info timed out. (connect timeout=60)'))
- [ ] borgzelle  ||  https://borgzelle.borg/  ||  HTTPSConnectionPool(host='borgzelle.borg', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f92585a5a60>: Failed to establish a new connection: [Errno -2] Name or service not known'))
- [ ] broadcity  ||  http://broadcity.in/  ||  Bad HTTP response code: 301
- [x] btdb  ||  https://btdb.unblockit.dev/  ||  Bad HTTP response code: 503
- [x] btdigg  ||  https://btdig.com/  ||  Bad HTTP response code: 503
- [x] btschool  ||  https://pt.btschool.club/  ||  Bad HTTP response code: 503
- [x] cinecalidad  ||  https://www.cinecalidad.is/  ||  Bad HTTP response code: 502
- [x] cinecalidad  ||  https://www.cinecalidad.eu/  ||  Bad HTTP response code: 502
- [x] cooltorrent  ||  http://cool-torrent.pl/  ||  HTTPConnectionPool(host='cool-torrent.pl', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7f925993d760>, 'Connection to cool-torrent.pl timed out. (connect timeout=60)'))
- [x] cpasbien  ||  https://wwwv.cpasbien-fr.fr/  ||  Bad HTTP response code: 403
- [x] dicmusic  ||  https://dicmusic.club/  ||  Bad HTTP response code: 301
- [x] divxtotal  ||  https://www.divxtotal.one/  ||  Bad HTTP response code: 503
- [x] epizod  ||  https://wwv.epizod.tv/  ||  Bad HTTP response code: 403
- [x] epublibre  ||  https://epublibre.org/  ||  Bad HTTP response code: 502
- [x] extratorrent-it  ||  https://extratorrent.unblockit.dev/  ||  Bad HTTP response code: 302
- [x] extratorrent-it  ||  https://extratorrent2.unblockninja.com/  ||  Bad HTTP response code: 301
- [x] eztv  ||  https://eztv.tf/  ||  Bad HTTP response code: 502
- [x] eztv  ||  https://eztv.root.yt/  ||  Bad HTTP response code: 301
- [x] fullmixmusic  ||  https://fullmixmusic.org/  ||  Bad HTTP response code: 303
- [x] hachede  ||  https://hachede.me/  ||  HTTPSConnectionPool(host='hachede.me', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f92585a5f40>: Failed to establish a new connection: [Errno 111] Connection refused'))
- [ ] hdonly  ||  https://hd-only.org/  ||  Bad HTTP response code: 500
- [ ] hdarea  ||  https://www.hdarea.co/  ||  Bad HTTP response code: 503
- [ ] hdchina  ||  https://hdchina.org/  ||  Bad HTTP response code: 302
- [ ] hdhome  ||  https://hdhome.org/  ||  Bad HTTP response code: 302
- [x] hdstreet  ||  https://hdstreet.club/  ||  Bad HTTP response code: 302
- [ ] hdtorrentsit  ||  https://hdtorrents.xyz/  ||  Bad HTTP response code: 404
- [x] hdu  ||  https://pt.upxin.net/  ||  Bad HTTP response code: 302
- [x] hdzone  ||  https://hdzone.me/  ||  Bad HTTP response code: 302
- [x] ilcorsaroblu  ||  https://ilcorsaroblu.org/  ||  HTTPSConnectionPool(host='ilcorsaroblu.org', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f9259a1be20>: Failed to establish a new connection: [Errno 111] Connection refused'))
- [x] kickasstorrent-kathow  ||  https://kickass-kat.com/  ||  Bad HTTP response code: 302
- [x] kickasstorrents-to  ||  https://kat.root.yt/  ||  Bad HTTP response code: 301
- [x] lastfiles  ||  http://last-torrents.org/  ||  Bad HTTP response code: 522
- [x] limetorrents  ||  https://www.limetorrents.info/  ||  Bad HTTP response code: 502
- [x] limetorrents  ||  https://www.limetorrents.asia/  ||  Bad HTTP response code: 502
- [x] limetorrents  ||  https://www.limetorrents.co/  ||  Bad HTTP response code: 502
- [x] limetorrents  ||  https://limetor.com/  ||  Bad HTTP response code: 502
- [x] limetorrents  ||  https://www.limetor.pro/  ||  Bad HTTP response code: 502
- [x] limetorrents  ||  https://limetorrents.unblockit.dev/  ||  Bad HTTP response code: 503
- [x] mactorrents  ||  https://mac-torrents.io/  ||  Bad HTTP response code: 503
- [x] monova  ||  https://monova.org/  ||  Bad HTTP response code: 502
- [x] monova  ||  https://monova.to/  ||  Bad HTTP response code: 502
- [x] mypornclub  ||  https://myporn.club/  ||  Bad HTTP response code: 503
- [x] onlyscene  ||  https://www.onlyscene.org/  ||  HTTPSConnectionPool(host='www.onlyscene.org', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f925993d370>: Failed to establish a new connection: [Errno 111] Connection refused'))
- [ ] oshenpt  ||  https://www.oshen.win/  ||  Bad HTTP response code: 302
- [x] prostylex  ||  https://prostylex.org/  ||  Bad HTTP response code: 503
- [x] psytorrents  ||  https://psytorrents.info/  ||  Bad HTTP response code: 404
- [x] thepiratebay  ||  https://thepiratebay.org/  ||  Bad HTTP response code: 502
- [x] thepiratebay  ||  https://pirateproxy.dev/  ||  Bad HTTP response code: 301
- [x] thepiratebay  ||  https://tpb.sadzawka.tk/  ||  Bad HTTP response code: 502
- [x] thepiratebay  ||  https://www.tpbay.win/  ||  Bad HTTP response code: 502
- [x] thepiratebay  ||  https://piratenow.xyz/  ||  Bad HTTP response code: 502
- [x] thepiratebay  ||  https://piratesbaycc.com/  ||  Bad HTTP response code: 502
- [x] torlock  ||  https://www.torlock.com/  ||  Bad HTTP response code: 502
- [x] torrentdownloads  ||  https://www.torrentdownloads.me/  ||  Bad HTTP response code: 502
- [ ] torrentccf  ||  https://et8.org/  ||  Bad HTTP response code: 302
- [x] torrentfunk  ||  https://www.torrentfunk.com/  ||  Bad HTTP response code: 502
- [x] torrentgalaxyorg  ||  https://torrentgalaxy.mx/  ||  Bad HTTP response code: 502
- [x] torrentgalaxyorg  ||  https://torrentgalaxy.root.yt/  ||  Bad HTTP response code: 301
- [x] torrentkitty  ||  https://www.torrentkitty.tv/  ||  Bad HTTP response code: 503
- [x] torrentkitty  ||  https://www.torrentkitty.se/  ||  Bad HTTP response code: 503
- [x] torrentkitty  ||  https://www.torrentkitty.io/  ||  Bad HTTP response code: 503
- [x] torrentkitty  ||  https://www.torrentkitty.vip/  ||  Bad HTTP response code: 503
- [x] torrentkitty  ||  https://www.torrentkitty.app/  ||  Bad HTTP response code: 503
- [x] torrentz2  ||  https://torrentz.unblockit.dev/  ||  Bad HTTP response code: 503
- [x] uniondht  ||  https://uniondht.org/  ||  Bad HTTP response code: 502
- [x] yggcookie  ||  https://www.yggtorrent.si/  ||  Bad HTTP response code: 503
- [x] yggtorrent  ||  https://www.yggtorrent.si/  ||  Bad HTTP response code: 503
- [x] yts  ||  https://yts.mx/  ||  Bad HTTP response code: 502
- [x] yts  ||  https://yts.root.yt/  ||  Bad HTTP response code: 301
- [x] zooqle  ||  https://zooqle.com/  ||  Bad HTTP response code: 502

**Comment by ilike2burnthing** [comment]
**Working - no action needed, but FYI**
anidex - uses DDoS-Guard https://github.com/FlareSolverr/FlareSolverr/issues/49
cpasbien - region locked
divxtotal - uses Cloudflare https://github.com/Jackett/Jackett/pull/10581
epizod - has had issues previously https://github.com/FlareSolverr/FlareSolverr/issues/25
mypornclub - uses Cloudflare https://github.com/Jackett/Jackett/pull/10513
prostylex - uses Cloudflare https://github.com/Jackett/Jackett/pull/10513
torrentkitty - all use Cloudflare https://github.com/Jackett/Jackett/pull/10513
yggcookie/torrent - uses Cloudflare https://github.com/Jackett/Jackett/pull/10513
_Those checked but not mentioned are working without issues_

**Working - main URL doesn't use Cloudflare, but proxies do - do we add a note?**
1337x - official proxies
https://audiobookbay.unblockit.dev/
https://btdb.unblockit.dev/
limetorrents - official proxies and unblockit.dev proxy
https://torrentz.unblockit.dev/

**Site is up, but I don't have an account to test - FlareSolverr notes may be needed**
borgzelle - no public DNS, IP must be set in hosts https://github.com/Jackett/Jackett/issues/9855
broadcity - uses CloudBunny DDoS protection
btschool - uses Cloudflare
dicmusic - redirects to https://kshare.club/
hdonly
hdarea - uses Cloudflare
hdchina
hdhome
hdtorrentsit - region locked
hdu
oshenpt
torrentccf

**Covered by other open issues**
3evils - down https://github.com/Jackett/Jackett/issues/4264
ats4u - needs to be changed https://github.com/Jackett/Jackett/issues/10578
bigtower - down https://github.com/Jackett/Jackett/issues/1099
btdigg - needs reworked? https://github.com/Jackett/Jackett/issues/10579
cooltorrent - down https://github.com/Jackett/Jackett/issues/9105
fullmixmusic - down https://github.com/Jackett/Jackett/issues/1034
hachede - down https://github.com/Jackett/Jackett/issues/10475
hdstreet - down https://github.com/Jackett/Jackett/issues/8450
onlyscene - down https://github.com/Jackett/Jackett/issues/8763
psytorrents - down https://github.com/Jackett/Jackett/issues/1152

**To Do**
https://1337x.unblocked.rest/ - remove, down (504)
https://kickass-kat.com/ - redirects to https://kick4ss.com/ - working
mactorrents  - add FlareSolverr note
https://pirateproxy.dev/ - redirects to https://pirateproxy.tube/ - working

root.yt - dead, remove all related proxies https://github.com/Jackett/Jackett/pull/7759#issuecomment-748627073

**Comment by ngosang** [comment]
> Working - main URL doesn't use Cloudflare, but proxies do - do we add a note?

I think it is not necessary to add the note in so many places. Only if FlareSolverr is required in all links.

> Site is up, but I don't have an account to test - FlareSolverr notes may be needed

I hope they reply in a reasonable time. I don't like having so many open issues. :smile:

**Comment by ilike2burnthing** [comment]
Many of those indexers covered by other open issues will likely be removed in a week. If there's no word back by then from those tagged above, just close this with the assumption that everything is working fine?

**Comment by ngosang** [comment]
It saddens me to see so few responses when asking for help.
I think everything is fine for now. In the future I will open another issue with an updated list.

**Comment by ilike2burnthing** [comment]
Just checked the accounts of all those tagged, there doesn't seem to be any activity in the last couple of months (or last couple of years for one of them), and as we know garfield is on a break. So at least we're not being ignored 😋

**Comment by RoloSoze** [comment]
@ngosang @ilike2burnthing I just got the chance to open my email & go over the notifications. I just ticked all the ones to which I have access & can confirm that they work fine in Jackett, with exception of HDChina because it's currently down but I checked it anyways because it was working fine until it went down a few days ago.
